src/docs/src/translation/id.json

```diff
@@ -0,0 +1,329 @@
+  "Modal is used to show a dialog or a box when you click a button.": "Modal is used to show a dialog or a box when you click a button.",
+  "Navbar is used to show a navigation bar on the top of the page.": "Navbar is used to show a navigation bar on the top of the page.",
+  "Pagination is a group of buttons that allow the user to navigate between a set of related content.": "Pagination is a group of buttons that allow the user to navigate between a set of related content.",
+  "Progress bar can be used to show the progress of a task or to show the passing of time.": "Progress bar can be used to show the progress of a task or to show the passing of time.",
+  "Radial progress can be used to show the progress of a task or to show the passing of time.": "Radial progress can be used to show the progress of a task or to show the passing of time.",
+  "Radio buttons allow the user to select one option from a set.": "Radio buttons allow the user to select one option from a set.",
+  "Range slider is used to select a value by sliding a handle.": "Range slider is used to select a value by sliding a handle.",
+  "Rating is a set of radio buttons that allow the user to rate something.": "Rating is a set of radio buttons that allow the user to rate something.",
+  "Select is used to pick a value from a list of options.": "Select is used to pick a value from a list of options.",
+  "Stack visually puts elements on top of each other.": "Stack visually puts elements on top of each other.",
+  "Stat is used to show numbers and data in a box.": "Stat is used to show numbers and data in a box.",
+  "Steps can be used to show a list of steps in a process.": "Steps can be used to show a list of steps in a process.",
+  "Swap allows you to toggle the visibility of two elements using a checkbox or a class name.": "Swap allows you to toggle the visibility of two elements using a checkbox or a class name.",
+  "Tabs can be used to show a list of links in a tabbed format.": "Tabs can be used to show a list of links in a tabbed format.",
+  "Table can be used to show a list of data in a table format.": "Table can be used to show a list of data in a table format.",
+  "Textarea allows users to enter text in multiple lines.": "Textarea allows users to enter text in multiple lines.",
+  "Toggle is a checkbox that is styled to look like a switch button.": "Toggle is a checkbox that is styled to look like a switch button.",
+  "Tooltip can be used to show a message when hovering over an element.": "Tooltip can be used to show a message when hovering over an element.",
+  "Code mockup": "Code mockup",
+  "Phone mockup": "Phone mockup",
+  "Window mockup": "Window mockup",
+  "Text Input": "Text Input",
+  "Range slider": "Range slider",
+  "Tabs": "Tabs",
+  "All daisyUI components": "All daisyUI components",
+  "Class name": "Class name",
+  "Type": "Type",
+  "Search": "Search",
+  "daisyUI themes": "daisyUI themes",
```

Comment:
are these intentionally left untranslated?

Reply:
I'd probably skip this section because I translate the documentation per page.
I'll translate them in the next commit.